doc/website/_get-started/02-installation.md

```diff
@@ -0,0 +1,35 @@
+---
+title: "Setup & Install"
+permalink: /get-started/setup/
+excerpt: "Instructions for installing the theme for new and existing Jekyll based sites."
+last_modified_at: 2022-12-05T21:36:18-04:00
+toc: true
+---
+
+## Getting started
+Follow the instruction at [https://dart.dev/get-dart](https://dart.dev/get-dart) to install `dart` in your environment.
+
+Once you have Dart installed, if you don't already have a project, you can create one using `dart create`: [https://dart.dev/tools/dart-tool](https://dart.dev/tools/dart-tool)
+
+Then add ROHD as a dependency to your pubspec.yaml file.  ROHD is [registered](https://pub.dev/packages/rohd) on pub.dev.  The easiest way to add ROHD as a dependency is following the instructions here https://pub.dev/packages/rohd/install.
```

Comment:
The install link should be an actual clickable link (both in README and in this new MD file)

Reply:
Fixed!